### DISCUSSION TITLE
Duty Cycle massivly increases after update to 3.77.7.20240826

### DISCUSSION THREAD

**Discussion by mk176** [OpenCCU/OpenCCU#2846] [🇺🇸 Q&A] [answered]
### Describe the issue you are experiencing

Duty Cycle of CCU3 and my Lan-Gateway (also Rasperrymatic) increases massivly after the update.
After 10 minutes after reboot the duty cycle of the CCU3 was 30% and of the Gateway 50%.
After another 30 min the duty cycle of the gateway reaches 100%.
I returned to version 3.77.6.20240720
Now duty cycle is 22% for CCU and 7% for Lan-Gateway after 45min.

### Describe the behavior you expected

Not such an increase of the duty cycle

### Steps to reproduce the issue

1. upgrade to version 3.77.7.20240826 (CCU is enougth)
2. Watch duty cycle of the LAN-Gateway
3.
...

### What is the version this bug report is based on?

3.77.7.20240826 

### Which base platform are you running?

rpi3 (RaspberryPi3, ARM64/aarch64)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
found nothing in the log, sorry
```

**Comment by jens-maus** [comment, marked as answer]
Sorry, this is not a bug or issue in RaspberryMatic itself, but simply a result of your own installation and setup of RaspberryMatic. Especially if there are excessive programs which perform certain RF communications upon RaspberryMatic restart end up in these kind of common DutyCycle situations. So please try to optimize your programs in not producing excessive RF communications.

**Reply by georgekue** [reply]
So what changed with the named release? I can confirm the very same behaviour: updated yesterday; before the update, the Duty Cycle usually was below 2% to 4%, now it is at about 10%. I just updated, no changes to the installation or setup of RaspberryMatic. Also, no firmware downloads are running.
